File: Java/Size_Empty.java
public int getSize() {
    int size = 0;
    for (int i = 0; i < TABLE_SIZE; i++) {
        KeyValue current = table[i];
        while (current != null) {
            size++;
            current = current.next;
        }
    }
    return size;
}

public boolean isEmpty() {
    for (int i = 0; i < TABLE_SIZE; i++) {
        if (table[i] != null) {
            return false; // Not empty
        }
    }
    return true; // Empty
}

File: C++/Size_Empty.cpp
int getSize() const {
    int size = 0;
    for (int i = 0; i < TABLE_SIZE; i++) {
        KeyValue* current = table[i];
        while (current) {
            size++;
            current = current->next;
        }
    }
    return size;
}

bool isEmpty() const {
    for (int i = 0; i < TABLE_SIZE; i++) {
        if (table[i] != nullptr) {
            return false; // Not empty
        }
    }
    return true; // Empty
}
